examples/telescope/bufcodegen/cosmos/app/v1alpha1/module_pb.ts

```diff
@@ -0,0 +1,204 @@
+// @generated by protoc-gen-es v1.2.0 with parameter "target=ts"
```

Comment:
this `bufcodegen` folder is still in the repo, is it needed?

Reply:
I believe it's been removed till the last commit.